js/apps/system/_admin/aardvark/APP/react/src/views/graphV2/addGraph/ExampleGraphForm.tsx

```diff
@@ -0,0 +1,128 @@
+import { ExternalLinkIcon } from "@chakra-ui/icons";
+import {
+  Box,
+  Button,
+  Flex,
+  HStack,
+  Link,
+  Stack,
+  Text,
+  VStack
+} from "@chakra-ui/react";
+import React, { useState } from "react";
+import { mutate } from "swr";
+import { ModalFooter } from "../../../components/modal";
+import { getRouteForDB } from "../../../utils/arangoClient";
+
+const exampleGraphsMap = [
+  {
+    label: "Knows Graph",
+    name: "knows_graph"
+  },
+  {
+    label: "Traversal Graph",
+    name: "traversalGraph"
+  },
+  {
+    label: "k Shortest Paths Graph",
+    name: "kShortestPathsGraph"
+  },
+  {
+    label: "Mps Graph",
+    name: "mps_graph"
+  },
+  {
+    label: "World Graph",
+    name: "worldCountry"
+  },
+  {
+    label: "Social Graph",
+    name: "social"
+  },
+  {
+    label: "City Graph",
+    name: "routeplanner"
+  },
+  {
+    label: "Connected Components Graph",
+    name: "connectedComponentsGraph"
+  }
+];
+
+export const ExampleGraphForm = ({ onClose }: { onClose: () => void }) => {
+  const createExampleGraph = async ({ graphName }: { graphName: string }) => {
+    const data = await getRouteForDB(window.frontendConfig.db, "_admin").post(
+      `/aardvark/graph-examples/create/${graphName}`
+    );
+    return data.body;
+  };
+  const [isLoading, setIsLoading] = useState(false);
+
+  return (
+    <VStack spacing={4} align="stretch">
+      {exampleGraphsMap.map(exampleGraphField => {
+        return (
+          <HStack key={exampleGraphField.name}>
+            <Box w="210px">
+              <Text>{exampleGraphField.label}</Text>
+            </Box>
+            <Button
+              colorScheme="blue"
+              size="xs"
+              type="submit"
+              isLoading={isLoading}
+              onClick={async () => {
```

Comment:
Maybe this is to much logic to have in the template and can be abstracted in a handler.

Reply:
This is done ✅